examples/server.js

```diff
@@ -1,10 +1,8 @@
 /* eslint-disable import/no-unresolved */
-const express = require('express');
-const { json } = require('express');
-const express_static = require('express').static;
-const UUID = require('uuid').v4;
-const { config } = require('dotenv');
-const { WebApi } = require('smile-identity-core');
+import { express, static as express_static, json } from 'express';
+import { v4 as UUID } from 'uuid';
+import { config } from 'dotenv';
+import { WebApi } from 'smile-identity-core';
```

Comment:
this change will break node 12 compatibility

Reply:
okay, will revert to require